### ISSUE TITLE
Errors with file names [Help]

### ISSUE THREAD

**Issue by kathrynmk** [wyguo/ThreeDRNAseq#41]
This is my first time using the app and I keep getting the same errors that I do not understand how to fix. The first error I am getting is "The compressed quantification file name or its sub-file names include special symbols. Please uncompressed it and then compressed again to "zip" format". However I have changed my file names to not include any symbols outside of letters and numbers and I still get this error every time. The names are all abundance with a number (ex. abundance12). Is there something I need to change about these names or is this a glitch with the app? The second error I am getting says that "The quantification file names in the sample meta-table do not match to the quantification files" however I have checked them countless times and cannot see any differences in the names in my metadata file and the file names. The correct column is selected for the file names in step 2 so I am not sure what could be causing this. I have included screenshots that will hopefully be helpful. Thanks in advance!
<img width="401" alt="appscreenshot" src="https://user-images.githubusercontent.com/86447914/219392086-0c128e64-31fa-49cb-b477-777b11f60ba2.png">
<img width="379" alt="filenamescreenshot" src="https://user-images.githubusercontent.com/86447914/219392090-5e730d20-3597-4f6e-97ce-540dc10382ce.png">
<img width="360" alt="metadatafilescreenshot" src="https://user-images.githubusercontent.com/86447914/219392091-1e79ba40-b324-4670-9152-aa64320c50a2.png">

**Comment by wyguo** [comment]
Hi, 
Thanks for using our App. Your quanti_file_names format is wrong. Please read the user manual to reformat the files:
https://github.com/wyguo/ThreeDRNAseq/blob/master/vignettes/user_manuals/Transcript_quantification_using_Galaxy.md

**Comment by kathrynmk** [comment]
Hi thanks for the reply. Would you be able to elaborate a bit? The quant files were generated by kallisto in command line, not using galaxy, which is why they are .tsv files. I was under the impression from the Input Files section of the manual that .tsv files were accepted as inputs. Do I need to convert them to a different file type? If you are not referring to the file type and instead referring to the way they are named, then perhaps I am missing something because I am not seeing anything about naming them in the manual. I have tried putting the same names in the metadata file column including the ".tsv" at the end and that did not work either. If you are able to give me a bit more clarification that would be greatly appreciated!

**Comment by wyguo** [comment]
Hi,
If the quantifications were generated with comand line, your files and file names should be OK. Just a quick questions. (1) When you upload the quantification files, did you select command line and kallisto in the corresponding widget? (2) Did you use winRAR or 7-zip to zip the quantification files? The App currently can't take the zip files generated by 7-zip.
Best

**Comment by kathrynmk** [comment]
Yes I selected both of those options in the widget. I also did not use 7-zip to zip my folder, I used the built-in zip program for Windows 11.

**Comment by wyguo** [comment]
It is very strange. Woud you mind sending me the data. I could check by running the source code.

**Comment by moe184** [comment]
Guys was this sorted? I am facing a very similar problem. thank you

**Comment by rakeshkpb87** [comment]
@wyguo I am also facing a similar issue and getting error "Transcript quantification files are not found. Please double check: (1) whether you have uploaded the quantification compressed folder and (2) whether the compressed folder name includes special symbols, e.g. extra white space." I generated quantification files using the command line with salmon. I compressed the main folder (containing each sample quantification folder (named RK1_quant to RK24_quant)) into a .zip format. After uploading the data, I am getting the above message. In the meta-data file, I gave the same folder name for the respective sample. Please help me to troubleshoot this issue. Thanks.
<img width="1315" alt="image" src="https://github.com/user-attachments/assets/a653f1c9-02ae-4dbe-862f-92ceda2fd888">

**Comment by sirventemi** [comment]
Hello,
I keep having the following error when trying to upload my data:

“The quantification file names in the sample meta-table do not match the quantification files. Please double check whether (1) the quantification file names in the metadata table match the files in the quantification folder or (2) you have selected the correct quantification method in Step 1: (3).”

I have carefully checked the file names in my metadata CSV and the quantification files generated by Salmon, and I do not see any differences.
I generated the quantification files using Salmon via the command line and compressed them into a .zip archive using WinRAR.
Attached are screenshots of:

-The metadata file content
-The zipped quantification files
-The selected options in (3) at imput data

Could you please help me identify what might be causing this mismatch?
Thank you!

<img width="349" height="260" alt="Image" src="https://github.com/user-attachments/assets/f368c2f4-8dfb-4702-8b0a-633a6e374851" />
<img width="519" height="101" alt="Image" src="https://github.com/user-attachments/assets/45e0fdab-b9e0-4902-9708-a52769e4dc04" />
<img width="284" height="119" alt="Image" src="https://github.com/user-attachments/assets/f31f7c46-e47c-4128-9b59-66318117bb75" />
